docs/built-in-functions.rst

```diff
@@ -0,0 +1,237 @@
+.. index:: function, built-in;
+
+.. _built_in_functions:
+
+***********************
+Built in Functions
+***********************
+
+Viper contains a set amount of built in functions that would be timely and/or unachievable to write in Viper.
+
+.. _functions:
+
+Functions
+=========
+The first list dictates each argument and the second sublist lists the types allowed as input.
```

Comment:
The type information appears to be available in the signature annotations. Would it be possible to utilize that info?

Reply:
I just removed this line as I think it just creates more confusion then it's worth.